kairos-server/src/routes/fetch.rs

```diff
@@ -0,0 +1,43 @@
+use axum::{extract::State, Json};
+use axum_extra::routing::TypedPath;
+use tracing::instrument;
+use serde::Deserialize;
+use chrono::NaiveDateTime;
+use kairos_data::transaction::*;
+
+use crate::{state::ServerState, AppErr};
+
+#[derive(Deserialize, Debug)]
+pub struct QueryTransactionsPayload {
+    sender: Option<String>,
+    min_timestamp: Option<String>,
+    max_timestamp: Option<String>,
+    min_amount: Option<i64>,
+    max_amount: Option<i64>,
+    recipient: Option<String>,
+}
+
+#[derive(TypedPath)]
+#[typed_path("/api/v1/query_transactions")]
```

Comment:
I would suggest naming the route `api/v1/tranasactions`

Reply:
Sure